pkg/cmd/cli/uninstall/uninstall.go

```diff
@@ -104,50 +103,58 @@ func Run(ctx context.Context, client *kubernetes.Clientset, extensionsClient *ap
 		if ns.Status.Phase == corev1.NamespaceTerminating {
 			fmt.Printf("Velero installation namespace %q is terminating.\n", namespace)
 		} else {
-			err = client.CoreV1().Namespaces().Delete(ctx, ns.Name, metav1.DeleteOptions{})
-			if err != nil {
+			if err := kbClient.Delete(context.Background(), ns); err != nil {
 				errs = append(errs, errors.WithStack(err))
 			}
 		}
 	}
 
 	// rolebinding
 	crb := install.ClusterRoleBinding(namespace)
-	if err := client.RbacV1().ClusterRoleBindings().Delete(ctx, crb.Name, metav1.DeleteOptions{}); err != nil {
+	key = kbclient.ObjectKey{Name: crb.Name, Namespace: namespace}
+	if err := kbClient.Get(context.Background(), key, crb); err != nil {
 		if apierrors.IsNotFound(err) {
-			fmt.Printf("Velero installation clusterrolebinding %q does not exist, skipping.\n", crb.Name)
+			fmt.Printf("Velero installation rolebinding %q does not exist, skipping.\n", crb.Name)
```

Comment:
It's a `ClusterRoleBinding` by type (L113), so we should keep the print statement as it was and update the comments.